src/function_classes/retrieval.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import torch.distributions as D
+
+from core import FunctionClass
+
+class Retrieval(FunctionClass):
+
+    def _init_param_dist(self) -> D.Distribution:
+        return D.Categorical(torch.Tensor([1]))
```

Comment:
Would passing in the sequence length to the init of this function let us create random values to do K/V matching with?

Reply:
could you clarify more what you mean by this? we already choose random values in the sequence to do K/V matching with

Reply:
opened tiny PR into this branch to illustrate